simtools/simulator.py

```diff
@@ -713,27 +522,5 @@ def _get_runs_to_simulate(self, run_list=None, run_range=None):
 
         """
         if run_list is None and run_range is None:
-            if self.runs is None:
-                msg = "Runs to simulate were not given as arguments nor in config_data - aborting"
```

Comment:
no error message?

Reply:
Raise now a 'ValueError' for a zero length run list.